plugins/modules/cloud/misc/terraform.py

```diff
@@ -287,8 +408,10 @@ def main():
             force_init=dict(type='bool', default=False),
             backend_config=dict(type='dict', default=None),
             backend_config_files=dict(type='list', elements='path', default=None),
+            plan_id=dict(type='str'),
+            log_activity_folder=dict(type='path', default='./'),
+            file_name=dict(type='str', default='audit'),
```

Comment:
Does the default here mean that logging is enabled by default? That would be a breaking change. I'd rather make it optional, but not the new default.

Reply:
please yes. It means that logging is enabled by default. I will make it optional instead.